com.unity.netcode.gameobjects/Runtime/SceneManagement/SceneEventData.cs

```diff
@@ -754,7 +754,7 @@ public void Dispose()
         internal SceneEventData(NetworkManager networkManager)
         {
             m_NetworkManager = networkManager;
-            InternalBuffer = NetworkBufferPool.GetBuffer();
+            InternalBuffer = new NetworkBuffer(1024, 256);
```

Comment:
How often is this triggered ?  Is this a worry for GC ?

Reply:
For every NetworkSceneManager instance there will be two (2) SceneEventData instances for the lifetime of that NetworkSceneManager instance.  There is always 1 NetworkSceneManager instance per network session (host, server, or client).
So, it will allocate roughly 2048 bytes total when StartHost, StartServer, or StartClient are invoked.  One of the two NetworkBuffers could increase in size during client synchronization (i.e. player is just approved and connecting to a session) depending upon how much has to be synchronized.

Reply:
I was following till I got here.  If we're cleaning ourselves up properly now, can we not go back to just using the PooledNetworkBuffer?  If not, can we have a comment saying why SceneEventData needs its own NetworkBuffer?

Reply:
Could you hoist what you just commented @NoelStephensUnity into comments in the code?

Reply:
I think it's fine to not use the pooled versions in this case (fine either way, honestly)... 2 instances created upon the construction of NetworkManager basically has the same impact as getting 2 from the pool since the pool doesn't pre-allocate any at all.

Reply:
Well, it was actual a bit lazy of me to do it that way.  This has since been fixed to make SceneEventData use only what it needs and then give the buffer back to the pool when no longer neeed.